test/forms/declaration/LegalDeclarationSpec.scala

```diff
@@ -0,0 +1,101 @@
+/*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package forms.declaration
+
+import helpers.views.declaration.WarehouseIdentificationMessages
+import play.api.libs.json._
+import unit.base.UnitSpec
+
+class LegalDeclarationSpec extends UnitSpec with WarehouseIdentificationMessages {
```

Comment:
Why are you extending `WarehouseIdentificationMessages`?

Reply:
Because I forgot to delete from the Spec I copied to create this one.